deployments/local-minio/docker-compose.yml

```diff
@@ -0,0 +1,122 @@
+# Configurable Registries
+#    This compose file allows the composition to be used in multiple environments. Instead of
+#    using hard-coded strings to identify the names of images, variable interpolation is used
+#    to make the registry part of the name configurable at runtime.
+#
+#    If the REGISTRY environment variable is not set, the registry defaults to
+#    Docker Hub (docker.io).
+#
+#    If the REGISTRY variable is set, interpolation on the command line can be used to inject
+#    a registry's address when the Docker service is deployed. the format of the command is:
+#
+#        docker stack deploy -c <(REGISTRY=<registry-url> docker-compose config) cdr
+#
+#    where <registry-url> is changed to be the address of the target registry.
+# --------------------------------------------------------------------------------------------------
+# To deploy to a cloud:
+#    1. Build the images locally.
+#    2. Tag those images to include the target registry address.
+#    3. Push those images.
+#    4. From the cloud environment, execute:
+#
+#        docker stack deploy -c <(REGISTRY=<registry-url> docker-compose config) cdr
+#
+#
+version: '3.7'
+services:
+  minio:
+    image: minio/minio:RELEASE.2019-06-19T18-24-42Z
+    entrypoint: sh
+    hostname: minio
+    ports:
+    - "9000:9000"
+    environment:
+      MINIO_ACCESS_KEY: MINIOEXAMPLEACCESSKEY
+      MINIO_SECRET_KEY: MINIOEXAMPLESECRETKEY
+    networks:
+      - cdr_local-minio
+    command: -c "mkdir /data/ingest-quarantine && /usr/bin/minio server /data"
+  ingest:
+    image: ${REGISTRY:-docker.io}/cnxta/cdr-ingest:0.1.0-SNAPSHOT
+    hostname: cdr-ingest
+    ports:
+      - target: 8080
+        published: 9040
+        protocol: tcp
+    networks:
+      - cdr_local-minio
+    deploy:
+      restart_policy:
+        condition: any
+    secrets:
```

Comment:
❓do we still need the aws secrets for a local deployment?

Reply:
No, it doesn't need any secrets for a local deployment - it uses the dummy `./secrets/minio_secret.notsec` and `./secrets/minio_access.notsec` keys as the source files down on line 115 and 117. I wasn't sure if the `aws_s3_secret.sec` and `aws_s3_access.sec` filenames were hard-coded, so to be safe I kept all the target file names & variables the same.